templates/react/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "@bun-examples/react",
+  "version": "0.1.82",
+  "dependencies": {
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0",
+    "web-vitals": "^3.0.3"
+  },
+  "eslintConfig": {
+    "extends": [
+      "react-app"
+    ]
+  },
+  "devDependencies": {
+    "@types/react": "^18.0.37",
+    "@types/react-dom": "^18.0.11",
+    "bun-types": "canary",
+    "typescript": "latest"
+  },
+  "scripts": {
+    "predev": "NODE_ENV=development bun build ./src/index.jsx --outfile ./.build/bundle.js",
```

Comment:
This is strictly worse than the status quo with `bun dev`.

Reply:
Why not `bun dev` for development and `bun build` for release?

Reply:
- I couldn't get `bun dev` working, presumably because it relies on the old `bun bun` which is gone?
- this is temporary until there's a JS API, at which point we can move the build step into `dev.tsx`. Then that's temporary until we have `Bun.App`. 
- imo we should also move away from `bun dev` anyway. We should use the bundler at this point even if it loses HMR and is a little more duct-tapey imo